src/quo/components/links/internal_link_card/view.cljs

```diff
@@ -0,0 +1,14 @@
+(ns quo.components.links.internal-link-card.view
+  (:require [quo.components.links.internal-link-card.channel.view :as channel.view]
+            [quo.components.links.internal-link-card.community.view :as community.view]
+            [quo.components.links.internal-link-card.user.view :as user.view]
+            [quo.theme :as quo.theme]))
+
+(defn- internal-link-card
+  [{card-type :type :as args}]
```

Comment:
nit `args` -> `props` as we mostly use that. Args would imply there is many arguments being passed here, however we are just passing a single map of props for the component